hbase-common/src/main/java/org/apache/hadoop/hbase/io/ByteBufferListOutputStream.java

```diff
@@ -39,37 +41,28 @@
 public class ByteBufferListOutputStream extends ByteBufferOutputStream {
   private static final Logger LOG = LoggerFactory.getLogger(ByteBufferListOutputStream.class);
 
-  private ByteBufferPool pool;
+  private ByteBuffAllocator allocator;
```

Comment:
final?

Reply:
Sounds good, will address in HBASE-22612.